README.md

```diff
@@ -45,6 +46,16 @@ println(base32HexString)
 assert(data.contentEquals(base32HexString.decodeBase32HexToBytes()))
 ```
 
+##### [Base45][rfc9285]
```

Comment:
nit, please add a link for `rfc9285`, ref: [README#L78](https://github.com/diglol/encoding/blob/trunk/README.md?plain=1#L78)